pkg/proxy/node.go

```diff
@@ -57,7 +57,7 @@ func (n *NodePodCIDRHandler) OnNodeAdd(node *v1.Node) {
 	if !reflect.DeepEqual(n.podCIDRs, podCIDRs) {
 		klog.ErrorS(nil, "Using NodeCIDR LocalDetector mode, current PodCIDRs are different than previous PodCIDRs, restarting",
 			"node", klog.KObj(node), "newPodCIDRs", podCIDRs, "oldPodCIDRs", n.podCIDRs)
-		panic("Current Node PodCIDRs are different than previous PodCIDRs, restarting")
+		klog.FlushAndExit(klog.ExitFlushTimeout, 1)
```

Comment:
This will lost the exit message
```suggestion
                klog.InfoS("Current Node PodCIDRs are different than previous PodCIDRs, restarting")
		klog.FlushAndExit(klog.ExitFlushTimeout, 1)
```

Reply:
sure.

Reply:
It seems `klog.ErrorS` log this in the error level, should we log it another time?

Reply:
Ops, sorry, miss that.